### PR TITLE
Validate extraAU

### PR DESCRIPTION
**Pull request by samblackk**
Attached to https://github.com/astronomer/astronomer/issues/469
Please r/m

### CODE REVIEW

File: src/lib/deployments/validate/index.js [hunk 1]
```diff
@@ -4,19 +4,35 @@ import validateProperties from "./properties";
...
+  deploymentId,
+  deployment
+) {
   // Validate environment variables.
   validateEnvironment(args.env);
 
+  const { executors } = config.get("deployments");
+
+  // Get the executor on this deployment.
+  const executor = get(deployment, "config.executor", AIRFLOW_EXECUTOR_CELERY);
```
Comment (schnie) [inline src/lib/deployments/validate/index.js]: @samblackk I _think_ we're doing this check / fallback in multiple places now. Do you think it makes sense to add a new constant called like `AIRFLOW_EXECUTOR_DEFAULT` and have that defined as 
```
export const AIRFLOW_EXECUTOR_DEFAULT = AIRFLOW_EXECUTOR_CELERY
```
so when we switch to kubernetes/knative as default we only change it one place?

Then we could replace these fallbacks with the new one.

Comment (samblackk) [reply src/lib/deployments/validate/index.js]: Good idea!

**Comment by samblackk** [comment]
@schnie GTG

File: src/resolvers/mutation/update-deployment/index.js [hunk 2]
```diff
@@ -62,7 +61,12 @@ export default async function updateDeployment(parent, args, ctx, info) {
   });
 
   // Validate our args.
-  await validate(deployment.workspace.id, mungedArgs, args.deploymentUuid);
+  await validate(
```
Comment (schnie) [inline src/resolvers/mutation/update-deployment/index.js]: @samblackk wonder if we should trim `validate()` down to 3 args or even 2. `deployment.workspace.id`, `args.deploymentUuid` and `deployment` could all be derived from `deployment`. 

Whacha think?

Comment (samblackk) [reply src/resolvers/mutation/update-deployment/index.js]: Cleaned up in the lates commit

**Pull request merged**